code/modules/projectiles/ammunition/bullets.dm

```diff
@@ -8,20 +8,20 @@
 	caliber = "9mm"
 	projectile_type = /obj/item/projectile/energy/flash
 
+/obj/item/ammo_casing/c9mmh
```

Comment:
this isn't in proper hierarchy.
it should be 
/obj/item/ammo_casing/c9mm/hv
/obj/item/ammo_casing/c9mm/rubber

etc

Reply:
Failed to notice this, will resolve.